doc/connectivity/networking/api/lwm2m.rst

```diff
@@ -517,3 +518,6 @@ API Reference
 
 .. _LwM2M specification 1.0.2:
    http://openmobilealliance.org/release/LightweightM2M/V1_0_2-20180209-A/OMA-TS-LightweightM2M-V1_0_2-20180209-A.pdf
+
+.. _LwM2M specification 1.1.1:
+   http://openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/
```

Comment:
Just to clarify, linking here to a directory instead of specific file is intended here, right? I guess it makes sense, given that the specification is scattered across different files.

Reply:
Yes, you got it right.